README.md

```diff
@@ -169,6 +170,21 @@ This option is used by JetBrains Marketplace by default.
   * `no`: all internal API usages will be reported. This is the default value.
   * `jetbrains-plugins`: internal API usages by JetBrains plugins will not be reported.  
 
+* `-mute` will mute (ignore) a specified plugin problem.
+This will mute any kind of supported plugin problems — including plugin problems related to the plugin descriptor.
```

Comment:
```suggestion
Supported values:
- `ForbiddenPluginIdPrefix`,
- `TemplateWordInPluginId`,
- `TemplateWordInPluginName`

A comma-separated list of ignored problems is allowed, e.g.:
`-mute ForbiddenPluginIdPrefix,TemplateWordInPluginName`
```

I think we should show usage ASAP, then go into details like "why does it even exist?".

If you decide to apply this suggestion, then delete:
>    A comma-separated list of plugin problem identifiers is allowed.
> 
>    The following plugin problem identifiers are supported:
>
>    - `ForbiddenPluginIdPrefix`,
>    - `TemplateWordInPluginId`,
>    - `TemplateWordInPluginName`

at the bottom, of course.